packages/plugins/response-cache/src/plugin.ts

```diff
@@ -572,28 +649,34 @@ export function useResponseCache<PluginContext extends Record<string, any> = {}>
             console.warn(
               '[useResponseCache] Cache instance is not available for the context. Skipping cache lookup.',
             );
+            return;
```

Comment:
Add a test for this, it was clearly a bug, a return statement was missing. No idea why TS is not catching this bug...